PHPCI/Helper/Github.php

```diff
@@ -108,4 +108,30 @@ public function getRepositories()
 
         return $rtn;
     }
+
+    public function createPullRequestComment($repo, $pullId, $commitId, $file, $line, $comment)
+    {
+        $token = Config::getInstance()->get('phpci.github.token');
```

Comment:
This need to expose the github token in project, I think we can consider to check an ENV variable and if not set check for the config, maybe this improvement can be in b8framework (let it make *twelve factor app* compliant)